plugins/nodes/src/js/components/NodesTable.tsx

```diff
@@ -1,5 +1,10 @@
 import * as React from "react";
-import { Table, Column, SortableHeaderCell } from "@dcos/ui-kit/dist/packages";
+import {
+  Table,
+  Column,
+  SortableHeaderCell,
+  HeaderCell
+} from "@dcos/ui-kit/dist/packages";
```

Comment:
Do we need the `dist/packages` part here?

Reply:
If you ask me I would add even more `/table` but I haven't touched the import

Reply:
Yeah ok, I just did not review the last PR, so 🤷‍♂️